proto/viam/app/v1/billing.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+
+package viam.app.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/v1";
+
+service BillingService {
+  rpc GetCurrentMonthUsageSummary(GetCurrentMonthUsageSummaryRequest) returns (GetCurrentMonthUsageSummaryResponse);
+  rpc GetUnpaidBalance(GetUnpaidBalanceRequest) returns (GetUnpaidBalanceResponse);
+  rpc GetInvoiceHistory(GetInvoiceHistoryRequest) returns (GetInvoiceHistoryResponse);
+  rpc GetItemizedInvoice(GetItemizedInvoiceRequest) returns (GetItemizedInvoiceResponse);
+  rpc GetBillingSummary(GetBillingSummaryRequest) returns (GetBillingSummaryResponse);
+}
+
+message GetCurrentMonthUsageSummaryRequest {
+  string org_id = 1;
+}
+
+message CurrentMonthUsageSummary {
+  double cloud_storage_usage = 1;
+  double data_upload_usage = 2;
+  double data_egres_usage = 3;
+  double standard_compute_usage = 4;
+  double total_usage_with_discount = 5; // returns number with any discounts applied
+  double total_usage_without_discount = 6; // returns number without any discounts applied
+}
+
+message GetCurrentMonthUsageSummaryResponse {
+  double cloud_storage_usage = 1;
```

Comment:
essentially CurrentMonthUsageSummary - the former is used as part of the entire BillingSummary request that is retrieving info for the entire page.

Reply:
what's the reason for not just using `CurrentMonthUsageSummary` here?

Reply:
The reasoning for the duplicate here is because we probably don''t need all the sub-requests here because we in theory want to get all the data for the single page `GetBillingSummaryRequest`, but i just left it in here in case we want to make requests tied to billing components rather than the page. 

I agree for API code readability sake it better to not have duplicate message structures here, but for the client it's better to not have nested layering, especially if the top layer has a single key.

Reply:
updated!